lib/checks/aria/aria-errormessage-evaluate.js

```diff
@@ -55,13 +55,21 @@ function ariaErrormessageEvaluate(node, options, virtualNode) {
     }
 
     if (idref) {
+      if (!isVisible(idref)) {
```

Comment:
You'll want to check that it's visible to screen readers, not that it's on screen:

```suggestion
      if (!isVisible(idref, true)) {
```

Please add tests checking that off screen messages don't error, and that a message with aria-hidden on it does.